framework/src/main/java/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -158,7 +145,73 @@ private static Pattern anchored(String regex) {
     }
 
     private static boolean isMemberSelect(String s, FlowExpressionContext context) {
-        return parseMemberSelect(s) != null;
+        return parseMemberSelect(s, context) != null;
+    }
+
+    /**
+     * Matches a field access which contains a method call either in receiver or in remaining. First
```

Comment:
The method is named `parseMethodCall`, but it's documented to match a field access.
Furthermore, its return type is similar to `parseMemberSelect` and dissimilar to other occurrences of `parseMethodCall`.  Please rename this to indicate what does, which is to parse a field access (a MemberSelect).